src/style-spec/reference/v8.json

```diff
@@ -2733,7 +2733,7 @@
         }
       },
       "match": {
-        "doc": "Selects the output whose label value matches the input value, or the fallback value if no match is found. The input can be any expression (e.g. `[\"get\", \"building_type\"]`). Each label must be either:\n - a single literal value; or\n - an array of literal values, whose values must be all strings or all numbers (e.g. `[100, 101]` or `[\"c\", \"b\"]`). The input matches if any of the values in the array matches, similar to the `\"in\"` operator.\nEach label must be unique. If the input type does not match the type of the labels, the result will be the fallback value.",
+        "doc": "Selects the output whose label value matches the input value, or the fallback value if no match is found. The input can be any expression (e.g. `[\"get\", \"building_type\"]`). Each label must be unique, and must be either:\n - a single literal value; or\n - an array of literal values, whose values must be all strings or all numbers (e.g. `[100, 101]` or `[\"c\", \"b\"]`).\n\nThe input matches if any of the values in the array matches using `===`, similar to the `\"in\"` operator.\nIf the input type does not match the type of the labels, the result will be the fallback value.",
```

Comment:
While the `===` reference makes sense for Javascript developers, the Style spec is programming language independent and is used in our Mobile SDKs as well. Would `strict equality` be an adequate replacement?

Reply:
Absolutely: 6a7c755 Thank you @asheemmamoowala for making this more clear!